conans/client/cmake.py

```diff
@@ -40,7 +40,8 @@ def _generator(self):
                         '10': '10 2010',
                         '11': '11 2012',
                         '12': '12 2013',
-                        '14': '14 2015'}
+                        '14': '14 2015',
+                        '15': '15 2017'}
```

Comment:
I think you can still use it, just force settings for that package while installing (incomplete):
```bash
$ conan install -s compiler.version=15 -s Pkg:compiler.version=14 ...
```